pytorch_lightning/logging/comet.py

```diff
@@ -52,7 +52,7 @@ def any_lightning_module_function_or_hook(...):
 try:
     from comet_ml import Experiment as CometExperiment
     from comet_ml import OfflineExperiment as CometOfflineExperiment
-    from comet_ml.papi import API
+    from comet_ml.api import API
```

Comment:
make it a try/except ImportError since some people may use an older version which does not have `api` yet
```
try:
    from comet_ml.api import API
except ImportError:
    # For more information, see: https://www.comet.ml/docs/python-sdk/releases/#release-300
    from comet_ml.papi import API

Reply:
Good idea, I agree!